docs/tutorial/notebook/reading_a_dataset.ipynb

```diff
@@ -0,0 +1,373 @@
+{
```

Comment:
<p>Make this image smaller - the font size should match the font size in the text.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Done.</p>